types/mparticle__web-sdk/package.json

```diff
@@ -0,0 +1,4 @@
+{
+    "private": true,
+    "license": "Apache-2.0"
```

Comment:
Please remove; we do not support different licenses in this field.

Reply:
Got it.  I'll just remove the `package.json`, not sure why this is even here.

Thanks for the prompt PRs btw, first time PR-ing into DefinitelyTyped and seems like a pretty well oiled machine :)